pkg/html/urlutil/urls.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2025 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package urlutil
+
+import "net/url"
+
+func AmendURL(baseURL, key, value string) string {
+	return TransformURL(baseURL, key, func(_ []string) []string {
+		if value == "" {
+			return nil
+		}
+		return []string{value}
+	})
+}
+
+func DropParam(baseURL, key, value string) string {
```

Comment:
It is hard to understand why DropParam may need the value w/o reading the tests.
Please document how to use it.